### PR TITLE
Added LocalDate diff and add support

### PR DESCRIPTION
**Pull request by stephengoldbaum**

### CODE REVIEW

File: morphir/sdk/core/src/morphir/sdk/LocalDate.scala [hunk 1]
```diff
@@ -36,12 +37,43 @@ object LocalDate {
   val Dec: Month = java.time.Month.DECEMBER
```
Comment (AttilaMihaly) [inline morphir/sdk/core/src/morphir/sdk/LocalDate.scala:37]: Is this used?

**Review by AttilaMihaly — CHANGES_REQUESTED** [review]

File: morphir/sdk/core/src/morphir/sdk/LocalDate.scala [hunk 2]
```diff
@@ -36,12 +37,43 @@ object LocalDate {
...
+  val Tuesday: DayOfWeek = java.time.DayOfWeek.TUESDAY
   val Wednesday: DayOfWeek = java.time.DayOfWeek.WEDNESDAY
-  val Thursday: DayOfWeek  = java.time.DayOfWeek.THURSDAY
-  val Friday: DayOfWeek    = java.time.DayOfWeek.FRIDAY
-  val Saturday: DayOfWeek  = java.time.DayOfWeek.SATURDAY
+  val Thursday: DayOfWeek = java.time.DayOfWeek.THURSDAY
+  val Friday: DayOfWeek = java.time.DayOfWeek.FRIDAY
+  val Saturday: DayOfWeek = java.time.DayOfWeek.SATURDAY
+
+  def addDays(count: Long, date: LocalDate) =
```
Comment (DamianReeves) [inline morphir/sdk/core/src/morphir/sdk/LocalDate.scala:48]: Can we add the return type

File: morphir/sdk/core/src/morphir/sdk/LocalDate.scala [hunk 3]
```diff
@@ -36,12 +37,43 @@ object LocalDate {
...
+    date plusDays count
+
+  def addWeeks(count: Long, date: LocalDate) =
```
Comment (DamianReeves) [inline morphir/sdk/core/src/morphir/sdk/LocalDate.scala:51]: Return types here too.

**Comment by DamianReeves** [comment]
On a whole I'd like to see the return types added. 
In reality the numeric return types should point to the morphir numeric types.

File: morphir/sdk/core/src/morphir/sdk/LocalDate.scala (hunk 2 above)
Comment (stephengoldbaum) [reply morphir/sdk/core/src/morphir/sdk/LocalDate.scala:48]: Good point

File: morphir/sdk/core/src/morphir/sdk/LocalDate.scala (hunk 3 above)
Comment (stephengoldbaum) [reply morphir/sdk/core/src/morphir/sdk/LocalDate.scala:51]: Yup

**Review by AttilaMihaly — APPROVED** [review]

**Pull request merged**